package.json

```diff
@@ -8,22 +8,21 @@
   },
   "scripts": {
     "prepare": "husky install && yarn build",
-    "clean": "lerna run clean && lerna clean --yes && rm -rf node_modules",
+    "clean": "nx run-many --target=clean --all && rm -rf node_modules/ .nx-cache/",
     "lint": "eslint packages/ scripts/ --ext=js,mjs,ts",
     "format": "prettier --write '{packages,scripts}/**/*.{js,mjs,ts,json,md}'",
     "bundlesize": "bundlesize --config ./scripts/bundlesize/bundlesize.config.json",
-    "build": "lerna run build --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
-    "build:performance": "yarn run build:performance:components && yarn run build:performance:benchmarks",
-    "build:performance:components": "lerna exec --scope @lwc/perf-benchmarks-components -- yarn build",
-    "build:performance:benchmarks": "lerna exec --scope @lwc/perf-benchmarks -- yarn build",
-    "dev": "lerna run dev --parallel --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
+    "build": "nx run-many --target=build --all --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
+    "build:performance": "yarn build:performance:components && yarn build:performance:benchmarks",
+    "build:performance:components": "nx build @lwc/perf-benchmarks-components",
+    "build:performance:benchmarks": "nx build @lwc/perf-benchmarks",
+    "dev": "nx run-many --target=dev --all --parallel=999 --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
```

Comment:
Based on this [issue](https://github.com/lerna/lerna/issues/3198) it seems that nx sets the parallelism to 999 to accomplish the equivalent of `lerna run --parallel`.

I copied this number from the Github issue but I think it's overkill, I can swap it to a smaller number if we want to.

Reply:
Also, parallel refers to the [max number of parallel processes,](https://nx.dev/nx/affected#parallel) not the actual number that are running at a given time.